src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -82,6 +94,24 @@ <h3>Launch with</h3>
               ><mat-icon class="anvil-icon" svgIcon="anvil"></mat-icon> AnVIL &raquo;</a
             >
           </div>
+          <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'WDL'">
+            <a
+              mat-raised-button
+              [matTooltip]="
+                !(hasContent$ | async)
+                  ? 'The WDL has no content.  '
+                  : (hasFileImports$ | async)
+                  ? 'BioData Catalyst powered by Terra does not support file-path imports in WDL. It only supports http(s) imports.'
+                  : 'NHLBI BioData Catalyst powered by Terra'
+              "
+              target="_blank"
+              rel="noopener"
+              [disabled]="!(hasContent$ | async) || (hasFileImports$ | async)"
```

Comment:
I think it's repeated enough that this should be a new single variable `[disabled]="blah"`

Reply:
Changed